### ISSUE TITLE
Greyscale image output

### ISSUE THREAD

**Issue by TimChaouoft** [TadasBaltrusaitis/OpenFace#967]
**Describe the bug**
First of all, I would like the thank the developer for the wonderful program!
I have 300 greyscale images of headshots. I wanted to get their head pose information. When I ran ./bin/FaceLandmarkImg -f, 285 of the 300 pictures generated output, which is great!

However, there were 15 images that did not generate any output. 
Even if I tried to change the model/algorithm .txt, I still get 285 outputs (i.e., csv). 
Has anyone encountered similar problem?

One way I tried to solve it is that I ran ./bin/FeatureExtraction -fdir with these unreadable face images. Although I know this command is designed for videos..When I ran this, all outputs were generated as frames, including the previously not recognized 15 images when I do facelandmarklmg. However, the confidence remained low. Please see the screenshot of my feature extraction, the one with 0.47 confidence was the one that did not produce any output when using "face landmarklmg". I wonder if I could still use the information generated by this output? 

Is there anyway to get the rest of the 15 images' output?
I'm a student at UofT doing a project. I would really appreciate it if I can get a solution!
Thank you very much!
[tim.chao@mail.utoronto.ca]
<img width="803" alt="Screen Shot 2021-05-10 at 8 47 40 PM" src="https://user-images.githubusercontent.com/83991947/117742444-d2093780-b1d2-11eb-88a1-275cc45843c3.png">

**Screenshots**

**Desktop (please complete the following information):**
 - OS: [11.3]
 - Version [e.g. 22] I just installed Openface a week ago!

**Additional context**
Example images. R70 is the one that didn't produce any output if I run Facelandmarklmg
![R70](https://user-images.githubusercontent.com/83991947/117742592-185e9680-b1d3-11eb-84f3-46d130412f0e.jpg)
![R69](https://user-images.githubusercontent.com/83991947/117742593-18f72d00-b1d3-11eb-8390-0b18c50891ef.jpg)
![R71](https://user-images.githubusercontent.com/83991947/117742594-18f72d00-b1d3-11eb-8839-8193c7c618d0.jpg)
![R72](https://user-images.githubusercontent.com/83991947/117742595-18f72d00-b1d3-11eb-9828-89b24096dbe8.jpg)

**Comment by TadasBaltrusaitis** [comment]
>However, there were 15 images that did not generate any output.

I don't think this is related to images being grayscale, but I think it is more ot do with face detection not having worked on those images. One thing you can try is changing the face detector used internally by OpenFace, but this would require changing code and rebuilding (the change would be fairly minor though)
